pocs/scheduler/__init__.py

```diff
@@ -1 +1,69 @@
+import os
+
+from astropy import units as u
+
+import pocs.utils.logger as logger_module
+from pocs.scheduler.constraint import Altitude
+from pocs.scheduler.constraint import Duration
+from pocs.scheduler.constraint import MoonAvoidance
 from pocs.scheduler.scheduler import BaseScheduler  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import horizon as horizon_utils
+from pocs.utils import load_module
+
+
+def create_scheduler_from_config(config, logger=None, observer=None):
+    """ Sets up the scheduler that will be used by the observatory """
+
+    if not logger:
+        logger = logger_module.get_root_logger()
+
+    if 'scheduler' not in config:
+        logger.info("No scheduler in config")
+        return None
+
+    if not observer:
+        logger.info("No valid observation found.")
```

Comment:
```suggestion
        logger.info("No valid Observer found.")
```

An `Observer` is different from an `Observation`. The `Observer` is the telescope itself, which has knowledge of the timezone and location where it is sitting.